### PR TITLE
Some changes+fixes

### PR DESCRIPTION
**Pull request by kvopper**

### CODE REVIEW

File: decisions/Rhovanion.txt [hunk 1]
```diff
@@ -612,8 +612,36 @@ political_decisions = {
...
 					NOT = { has_pop_culture = dorwinrim }
 				}
-				militancy = 1
-				consciousness = 2
+				scaled_consciousness = {
+					factor = 6
+					OR = {
+						ideology = high_nobility
+						ideology = low_nobility
+					}
```
Comment (SirRunner) [inline decisions/Rhovanion.txt:620]: Does dropped an OR in here actually work? I've always split it into two scaled_consciousness blocks

**Review by SirRunner — CHANGES_REQUESTED** [review]

File: decisions/Rhovanion.txt (hunk 1 above)
Comment (kvopper) [reply decisions/Rhovanion.txt:620]: I was fully confident that it works and didn't check. Gonna test RN.

File: localisation/ffff.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    icon = 9
+    tax_efficiency = 0.4
+    administrative_efficiency_modifier = 0.3
+    tariff_efficiency_modifier = 0.3
+    education_efficiency_modifier = 0.05
+    supply_consumption = -0.9
+    poor_income_modifier = 1
+    middle_income_modifier = 3
+    rich_income_modifier = 5
+}
```
[the remark this answers is not in the document]
Comment (kvopper) [reply localisation/ffff.txt:20]: Oops I forgot to delete this file.

File: decisions/Rhovanion.txt (hunk 1 above)
Comment (kvopper) [reply decisions/Rhovanion.txt:620]: image

Oops I messed up brackets

**Review by SirRunner — APPROVED** [review]

**Pull request merged**